Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -759,34 +769,33 @@ final class StatusTableViewController: ChartsTableViewController {
             
             switch ChartRow(rawValue: indexPath.row)! {
             case .glucose:
-                cell.chartContentView.chartGenerator = { [weak self] (frame) in
+                cell.setChartGenerator(generator: { [weak self] (frame) in
                     return self?.statusCharts.glucoseChart(withFrame: frame)?.view
-                }
-                cell.titleLabel?.text = NSLocalizedString("Glucose", comment: "The title of the glucose and prediction graph")
+                })
+                cell.setTitleLabelText(label: NSLocalizedString("Glucose", comment: "The title of the glucose and prediction graph"))
```

Comment:
[nit] didn't catch this on the LoopKit side, but the `generator` and `label` argument labels seem extraneous at the call-site